.github/actions/tscommon/README.md

```diff
@@ -0,0 +1,24 @@
+### Links
+https://stackoverflow.com/questions/15806241/how-to-specify-local-modules-as-npm-package-dependencies
+https://github.com/bersling/typescript-library-starter/blob/master
+https://www.tsmean.com/articles/how-to-write-a-typescript-library/
+
+### Error and fix
+To avoid `npm ci` error:
+```shell
+npm ERR! code EUSAGE
+npm ERR! 
+npm ERR! `npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
+npm ERR! 
+npm ERR! Missing: tscommon@0.0.0 from lock file
+npm ERR! 
+npm ERR! Clean install a project
+```
+
+Use:
+```shell
+$ cd tscommon
+$ npm pack
+$ cd ../another-action
+$ npm install --save ../tscommon/tscommon-0.0.0.tgz
```

Comment:
Does this mean that we need to run `npm pack` in the `tscommon` directory once to create the `tscommon-0.0.0.tgz` in order to build other actions locally?

Reply:
that's correct. I did not need to do that on my local machine, I could just use `../tscommon`. On GitHub it failed. Adding the tarball as dependency added an `integrity: sh512-` entry to tscommon dependency in the lock file, and this fixed the issue.
But if there's another way to fix it, I'd like to use it.

Reply:
SG